### DISCUSSION TITLE
[Question] Ventra - Single Use

### DISCUSSION THREAD

**Discussion by TheDingo8MyBaby** [metrodroid/metrodroid#852] [Q&A]
I apologize for this being out of the blue, and probably not the best worded, but I just recently found out about MetroDroid.

Within the past 6 months, I've been trying to learn more about NFC/UHF/Etc...

I currently have a Proxmark3 and in one of the Discords I'm apart of (IceMan), I was able to find out about MetroDroid.

I've used my Proxmark to clone some cards here n there and also was able to find/report a vulnerability at a Semi-Local Driving Range, where they had the "Currency" setup as Plain ASCII on a Mifare Classic card.

Anyway, I currently have 2 Disposable 1 day use Ventra Tickets, along with a Ventra Card that is tied to my friend's Student ID.

What I'm currently curious about though, is when scanning the Ventra Disposable via MetroDroid. How are you able to determine the Expiration Date of the card?  

I tried to read through some of the documentation, but I'm kind of at a loss.  As to how MetroDroid is able to know the details of a card.  Like, how it knows that my card expires 01/26/2024 at midnight.  Or, how it knows how much is on the card.

I have attached a few screenshots of what I see when I scan the Card. 

But, I was wondering if someone might be able to help me better understand, how you are able to get this data?

I'm just genuinely intrigued by this, and would like a bit of help learning more, while I go down this rabbit hole.

Thank You in advance.

Also, if you want/need any Proxmark Output.  Let me know what you'd like me to run, and I'll get the info back to you.

![Screenshot_20240131-023106.png](https://github.com/metrodroid/metrodroid/assets/140236702/28564f74-5f27-4a46-8411-20e5036b22c4)

![Screenshot_20240131-023057.png](https://github.com/metrodroid/metrodroid/assets/140236702/1dedd7e1-3893-4b19-9b76-c4db3e78d763)

![Screenshot_20240130-223717.png](https://github.com/metrodroid/metrodroid/assets/140236702/ce2d6edb-73b7-4c0b-8e60-7ac6efb70857)

![Screenshot_20240130-223737.png](https://github.com/metrodroid/metrodroid/assets/140236702/03c3a174-6996-4b2c-95cc-1ee8e69a7898)

![Screenshot_20240131-023046.png](https://github.com/metrodroid/metrodroid/assets/140236702/9886e502-918f-474d-b9f4-268f7fcc551f)

**Comment by rom4ster** [comment]
I am by no means a expert but would that information be on the card? Generally designing these systems would simply put an ID on the card and then check the expiration server side
